app/controllers/api/v2/override_values_controller.rb

```diff
@@ -8,7 +8,7 @@ class OverrideValuesController < V2::BaseController
       before_action :find_override_values
       before_action :find_override_value, :only => [:show, :update, :destroy]
       # override return_if_smart_mismatch in LookupKeysCommonController to add :index, :create
-      before_action :return_if_smart_mismatch, :only => [:index, :create, :show, :update, :destroy]
+      before_action :return_if_smart_mismatch, :only => [:create, :show, :update, :destroy], :except => [:index]
```

Comment:
I think you do not need the `:except`, removing the `:index` from array for `:only` is enough. The comment above it will also need a change to keep it consistent

Reply:
thanks for reviewing this PR, verified the suggestions and updated the PR.